src/content/api/asset-transfers.mdx

```diff
@@ -0,0 +1,361 @@
+| created  | Asset transfer successfully created                                                                                |
+| sent     | Asset transfer notification (sms, email) was sent to a new user                                                    |
+| expired  | Asset transfer expired as new user didn't create his account and claimed the asset. This is very short lived state |
+| returned | Asset transfer expired and was returned to original owner                                                          |
+| claimed  | Asset transfer was successfully completed                                                                          |
+
+---
+
+
+<Endpoint
+  method="POST"
+  path="/api/asset-transfers"
+>
+  ## Create an Asset Transfer
+
+  Transfer an asset to a recipient. Some assets can be transfered only in whole
+  (eg giftcards or tokens) while others can be transfered only in part (eg money).
+
+  Some assets can be transferred without supplying a recipient. A `url` field will be
+  returned in these cases. The `url` will link to a page to claim the asset.
+
+  ### Attributes
+  <Properties>
+    <Property name="assetId" type="string" required>
+      Id of a discrete asset to transfer or wallet to draw from.
+    </Property>
+
+    <Property name="recipientAlias" type="string">
+      Phone number, email or handle of receiver.
+    </Property>
+
+    <Property name="description" type="string">
+      Shows up in transaction history against the transfer. 200 character limit.
+    </Property>
+
+    <Property name="message" type="string">
+      Shows up in transaction history against the transfer. 100 character limit.
+    </Property>
+
+    <Property name="value" type="bignumber">
+      Amount to send. Required for money transfers. Units depend on the asset type.
+    </Property>
+
+    <Property name="senderName" type="string">
+      Human readable name for the sender. 30 character limit.
+    </Property>
+
+    <Property name="suppressNotification" type="boolean">
+      Shows up in transaction history against the transfer.
+    </Property>
+  </Properties>
+
+  ### Errors
+  <Properties>
+    <Error code="403" message="INSUFFICIENT_WALLET_BALANCE">
+      The value of the asset-transfer exceeds the balance on the wallet.
+    </Error>
+
+    <Error code="403" message="QUOTA_EXCEEDED">
+      The transfer exceeds one or more spend quota limits. See [Quota Error Response](https://docs.centrapay.com/api/quotas#quota-error-response).
+    </Error>
+
+    <Error code="403" message="ASSET_NOT_ACTIVE">
+      The asset is not active and cannot be transferred.
+    </Error>
+
+    <Error code="403" message="RECIPIENT_MISSING">
+      The asset must be transferred with a recipient supplied.
+    </Error>
+  </Properties>
+
+  <CodePanel slot="code-examples" title="Whole Asset Transfer" method="POST" path="/api/assets/asset-transfers">
```

Comment:
@AndyClifford I believe the decision was just to have one request and one response example per endpoint. If we need multiple examples it's a sign our guides should be improved. IMO you should pick the most relevant one and just use that.

Reply:
@AndyClifford I just realised I was tagging you but it's Alex's PR - sorry 😂